src/utils/component-utils.js

```diff
@@ -27,3 +27,14 @@ export function getCurrentStreamer() {
 
   return null;
 }
+
+export function getPlayerInfo(sessionId) {
+  if (!window.APP || !window.APP.componentRegistry || !window.APP.hubChannel || !window.APP.hubChannel.presence)
```

Comment:
I think I got this from:

https://github.com/mozilla/hubs/blob/749c7f4c0b4bd86aa40f74d478531078ceb54223/src/utils/component-utils.js#L14

Not sure why we need that there  but I kept it. I guess in this case we should never call this method if there is not a hub channel so probably it can be removed.